solutions.js

```diff
@@ -1,5 +1,24 @@
-console.log('Defining functions!');
+// console.log('Defining functions!');
+//
+// function changeNavColor() {
+//   // change the navbar to a different color
+// };
 
-function changeNavColor() {
-  // change the navbar to a different color
-};
+1.
+function changeColor (){
+    topbar.style.backgroundColor = 'red';
+
+}
+changeColor();
+
+2.
+var newElement = document.querySelector('h1');
+    newElement.prepend("This page is now under Andrew\'s control!");
+
+// var newElement = document.querySelector('h1');
+//     newElement.style.fontSize = '18px';
+//     newElement.prepend("This page is now under Andrew\'s control!");
+
+3.
+var topHalf = document.querySelector('.cards');
```

Comment:
This will select ALL the cards, how do you select only half to apply the style display:none?